From b8b4321a5afbdd5ac959061171d57f132298c833 Mon Sep 17 00:00:00 2001
From: Russell Webb <russell.webb@intel.com>
Date: Fri, 13 Dec 2013 10:03:56 -0800
Subject: Update houdini for mixin setting

This property must be set whenever houdini is enabled to inform
the frameworks that we do support the arm-v7 instruction set.

Change-Id: Ib1c752291588f24449dc30820ac4f0040337bc88
---
 houdini.mk |   20 ++++++++++++++++++++
 1 files changed, 20 insertions(+), 0 deletions(-)

diff --git a/houdini.mk b/houdini.mk
index a7047d0..7be6721 100644
--- a/houdini.mk
+++ b/houdini.mk
@@ -1,3 +1,21 @@
+#Houdini prebuilt
+HOUDINI_ARM_PREBUILTS_DIR := vendor/intel/houdini/arm
+houdini_prebuilt_stamp := $(HOUDINI_ARM_PREBUILTS_DIR)/stamp-prebuilt-done
+houdini_prebuilt_done := $(wildcard $(houdini_prebuilt_stamp))
+
+ifneq ($(houdini_prebuilt_done),)
+INTEL_HOUDINI := true
+#Houdini
+PRODUCT_PACKAGES += libhoudini \
+    houdini \
+    enable_houdini \
+    disable_houdini \
+    check.xml \
+    cpuinfo \
+    cpuinfo.neon
+
+#houdini arm libraries
+
 ### Extra files needs to copy for houdini
 HOUDINI_PATH := vendor/intel/houdini
 #"Copying Houdini arm libs"
@@ -92,3 +110,5 @@ PRODUCT_COPY_FILES += \
 PRODUCT_COPY_FILES += \
     ${HOUDINI_PATH}/system/lib/arm/cpuinfo:system/lib/arm/cpuinfo \
     ${HOUDINI_PATH}/system/lib/arm/cpuinfo.neon:system/lib/arm/cpuinfo.neon \
+
+endif
-- 
1.7.4.1

